docs/source/migration.mdx

```diff
@@ -1133,9 +1133,13 @@ const server = new ApolloServer({
 const result = await server.executeOperation({
   query: 'query helloContext { hello }',
 }, {
-  // A half-hearted attempt at making something vaguely like an express.Request,
-  // and not bothering to make the express.Response at all.
-  req: { headers: { name: 'world' } },
+  // highlight-start
+  contextValue: {
+    // A half-hearted attempt at making something vaguely like an express.Request,
```

Comment:
This change is the AS3 version, so we shouldn't change it. (The comment is trying to point out that it was actually kind of hard to use the AS3 version since you theoretically had to come up with a whole express.Request.)